clients/linux/src/app.rs

```diff
@@ -841,7 +841,12 @@ impl Gui {
 
         // Window.
         let w = GtkAppWindow::new(app);
+        let icon = GdkPixbuf::from_inline(LOGO_INTRO, true)
```

Comment:
nit: Let's declare the `icon` variable just above of the "block" of code dedicated the window as it's own thing.